server/game/cards/02.3-EDS/AllieHensmanExp1.js

```diff
@@ -0,0 +1,43 @@
+const DudeCard = require('../../dudecard.js');
+
+class AllieHensmanExp1 extends DudeCard {
+    setupCardAbilities(ability) {
+        this.job({
+            title: 'Allie Hensman (Exp.1)',
+            playType: 'noon',
+            cost: ability.costs.bootSelf(),
+            target: {
+                activePromptTitle: 'Mark an opposing dude',
+                cardCondition: {
+                    location: 'play area',
+                    controller: 'opponent'
+                },
+                cardType: ['dude']
+            },
+            message: context => this.game.addMessage('{0} has {1} go after {2}',
+                context.player, this, context.target),
+            onSuccess: (job, context) => {
+                const MarkWorth = job.mark.influence;

```

Comment:
Please use camelCase on var names, so they can avoid being mistaken for ClassNames